doc/_includes/tutorials/common/samples/galaxy-export.md

```diff
@@ -12,17 +12,15 @@ After you log in to IRIDA (or if you were already logged in), you will be direct
 
 ![Galaxy IRIDA projects list.]({{ site.baseurl }}/images/tutorials/common/samples/galaxy-irida-projects-list.png)
 
-Navigate to the list of samples that you're interested in exporting by clicking on the project name. Then, [select the samples]({{ site.baseurl }}/user/user/samples/#selecting-samples) that you want to export, click on the "Export" button just above the samples list, then click "Send to Galaxy":
```

Comment:
I noticed that the **Galaxy data library** link is broken at the top of this page. I realize it's not something that you touched, but could you update it to point to <https://galaxyproject.org/data-libraries/>?

Reply:
Updated in 00f7f59d6e7217541ac0b79043f776d22b282d38